### ISSUE TITLE
Using Tensorflow

### ISSUE THREAD

**Issue by JohnTroony** [ChunML/text-generator#6]
I'm trying to implement the recurrent neural network using Keras but with Tensorflow as the back-end instead of Theano. Is this possible? Any gotchas or tips/tricks are appreciated. Thank you!

**Comment by ashishk008** [comment]
Yes it is possible to change the backend of Keras. You can find the config file of keras at **/.keras/keras.json** .
The file looks something like this:

{    "epsilon": 1e-07,
    "floatx": "float32",
    "image_data_format": "channels_last",
    "backend": "tensorflow"
}
In your case the backend might be Theano. You can change it by replacing "theano" with "tensorflow".

**Comment by JohnTroony** [comment]
Thanks, I figured it out. I had a few dependencies missing.
